packages/react/src/components/TextField/TextField.types.ts

```diff
@@ -75,6 +75,12 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
    */
   autoAdjustHeight?: boolean;
 
+  /**
+   * Ref used to keep the current scrollTop position when having a scrollable wrapper. This helps avoid issues when
+   * deleting a line of text causing the container to scroll to the top.
+   */
```

Comment:
nit: for clarity
```suggestion
  /**
   * A read-only ref to the scrollable element that contains this TextField, if any. Only needed if 
   * `autoAdjustHeight` is set. This allows TextField to maintain the scroll position when text is edited.
   */
```

Also might want to mention this in the documentation for `autoAdjustHeight`
```diff
  /**
   * For multiline text fields, whether or not to auto adjust text field height.
+  * If inside a scrollable container, also set `scrollContainerRef`.
   * @defaultvalue false
   */
  autoAdjustHeight?: boolean;
```

Reply:
can i ask a question? i see the codes of this Pull Request has merged into master branch. 
but how can i know it will work in https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield.
that mean there are new version that have those codes to fix this issue.?

Reply:
Hi @NguyenVanBay101097, you can take a look at our changelog: https://github.com/microsoft/fluentui/blob/master/packages/react/CHANGELOG.md. You can see the top entry is 8.114.0 and it includes this PR. This changelog is automatically updated upon release.